wiki/items/food.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Adding food
+index: 2
+---
+# Adding food
+This tutorial will assume that You have understood [Creating your first item](first-item)
+
+An item can be made food by adding a `FoodComponent` to the Item Settings which you can create by using the Builder. Try reading the javadocs of the builderto understand whats going on here or continue reading for an explanation:
+
+```java
+public static final Item EXAMPLE_FOOD = new Item(new QuiltItemSettings().food(new FoodComponent.Builder()
```

Comment:
```suggestion
public static final Item EXAMPLE_FOOD = new Item(new QuiltItemSettings().food(
    new FoodComponent.Builder()
```
put `new FoodComponent.Builder()` on a new line so it more easily be matched up with the list explanation below